docs/internals/orientation-vector.md

```diff
@@ -3,41 +3,39 @@ title: "Orientation Vector"
 linkTitle: "Orientation Vector"
 weight: 10
 type: "docs"
-description: "A description of Viam's orientation vector method."
+description: "A description of Viam's orientation vector object."
 ---
-Viam's orientation vector is a method for describing the orientation of an object in 3D space.
-It is part of a Pose which also includes the position in 3D space.
 
-## Basics
+An _orientation vector_ specifies the angle at which an object is oriented in 3D space.
```

Comment:
"angle" is the wrong term here. Maybe "specifies the orientation of an object in 3d space"?